joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -112,4 +112,103 @@ class SimpleAstCreationPassTest extends RubyCode2CpgFixture {
       encoding.columnNumber shouldBe Some(5)
     }
   }
+
+  "Code field for simple fragments" should {
+
+    "have correct code for a single left had side call" in {
```

Comment:
Moreover, once moved to the previous group, tests should ideally be renamed as well (as we're not testing .code exclusively), to keep things aligned.